### PR TITLE
Adding examples of sandbox code

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

File: examples/erc20DividendSandbox.ts [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+  };
+
+  // Instantiate the API
+  const polymathAPI = new PolymathAPI(params);
+
+  // Get some poly tokens in your account and the security token
+  const web3Wrapper = new Web3Wrapper(params.provider, {
+    gasPrice: params.defaultGasPrice,
+  });
+  const address = await web3Wrapper.getAvailableAddressesAsync();
```
Comment (VictorVicente) [inline examples/erc20DividendSandbox.ts]: Use `await polymathAPI.getAccount()` instead

File: examples/erc20DividendSandbox.ts [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  await polymathAPI.getPolyTokens({ amount: new BigNumber(1000000), address: address[0] });
+  await polymathAPI.getPolyTokens({
```
Comment (VictorVicente) [inline examples/erc20DividendSandbox.ts]: Use `transfer` instead

File: examples/countTransferManager.ts [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+  const finalNames = resultNames.map(name => {
+    return bytes32ToString(name);
+  });
+  const index = finalNames.indexOf(moduleStringName);
+
+  // Call to add count transfer manager module with max 3 holders
+  await tickerSecurityTokenInstance.addModule({
+    moduleName,
+    address: modules[index],
+    maxCost: new BigNumber(100000),
```
[the remark this answers is not in the document]
Comment (victorwiebe) [reply examples/countTransferManager.ts]: I used setupcost for all modules

File: examples/erc20DividendSandbox.ts (hunk 2 above)
Comment (victorwiebe) [reply examples/erc20DividendSandbox.ts]: Based on this, does it mean that I should only getPolyTokens for polymathAPI.getAccount() ? In that any time the module, security token(registry) needs tokens we need to transfer them from our local account?

Comment (monitz87) [reply examples/erc20DividendSandbox.ts]: Yeah, it means exactly that. Tokens should be requested from the faucet only for the account that plans to later transfer them to another contract.

**Pull request merged**